app/controllers/groups_controller.rb

```diff
@@ -65,14 +59,15 @@ def group_params
   end
 
   def add_member
-    @group = Group.find(params[:id])
     unless current_user.owns_group?(@group)
       return render file: 'public/403.html',
                     status: :unauthorized
     end
 
     @user = User.find(params[:user_id])
     @group.members.append(@user) unless @group.members.include?(@user)
+    @notification = AddedToGroupNotification.new(receiver: @user, date: Time.zone.now, group_name: @group.name,
+                                                 active: false, unread: true).save
```

Comment:
Please do not send the notification if the user is already a member (expand the `unless` from the line above).